.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,34 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+          
+      - name: cppcheck_run
+        uses: deep5050/cppcheck-action@main
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN}}
+          std: c++11
+          max_ctu_depth: 4
+          output_file: cppcheck_run.log
+          #exclude all third-party libs but `realdds`
+          other_options: -ithird-party/easyloggingpp -ithird-party/glad -ithird-party/sqlite -ithird-party/imgui -ithird-party/glfw -ithird-party/hidapi
```

Comment:
Can we do `-ithird-party`?

Although I am not sure it's good since Eran insert our own utilities to third-party.
What do you think?

Reply:
I list the folders separately because we have some third-party folders that we do want to check, like realdds and utilities.